manga_translator/translators/vertex.py

```
@@ -0,0 +1,586 @@
+                        await asyncio.sleep(sleep_time)
+
+                if retry_attempt > 0 and current_temperature != self.temperature:
+                    self.logger.info(f"[重试] 温度调整: {self.temperature} -> {current_temperature}")
+
+                # 发送 HTTP 请求
+                url = self._build_api_url()
+                headers = {
+                    "Content-Type": "application/json",
+                }
+
+                # 始终打印请求详情（用于调试）
```

Ruff (0.14.13)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)